include/libtorrent/kademlia/observer.hpp

```diff
@@ -146,19 +144,11 @@ struct observer : boost::noncopyable
 
 	node_id m_id;
 
-	TORRENT_UNION addr_t
```

Comment:
is this change necessary? it will make the observer base class larger, and it's not obvious if there's any gain